SavingsManagementSystem.Service/Authentication/Implementations/AuthenticationService.cs

```diff
@@ -103,25 +95,23 @@ public async Task<LoginResponse> Login(LoginRequest loginRequest)
 			return response;
 		}
 
-		public async Task<string> ForgetPasswordAsync(string email)
+		public async Task ForgetPasswordAsync(string email)
 		{
 			var user = await _userManager.FindByEmailAsync(email);
 			if (user == null)
 			{
 				throw new ArgumentNullException($"Email {email} provided does not exist in our Database");
 			};
 
-			var vToken = await _vTokenService.CreateVerificationTokenAsync(user.Id, 30);
+			var vToken = await _vTokenService.CreateVerificationTokenAsync(30, "Successful", email, user.Id);
```

Comment:
send date instead of 30. DateTime.UtcNow.AddMinutes(30). and why are you sending "successful"

Reply:
that successful there is the status that will save in the database to confirm that the email was sent or not.
and for this, send date instead of 30. DateTime.UtcNow.AddMinutes(30). in the createVerificationToken, i have already added ExpiryMinute = DateTime.UtcNow.Addminutes(expiryMnutes).  so the minute is the one i am collecting here